cmd/tckedit.cpp

```diff
@@ -142,7 +142,7 @@ void run ()
 
   // Get the consensus streamline properties from among the multiple input files
   Tractography::Properties properties;
-  size_t count = 0;
+  size_t count = 0, total_count = 0;
```

Comment:
is this supposed to be merged to master?

Reply:
This depends on whether the same warnings pop up during CI... If they do, the CI will fail. But you're right, it's probably best to leave this alone on `master` unless the CI does fail. We can fix it on `dev` in the meantime...